apps/testingaccessibility/src/utils/format-prices-for-product.ts

```diff
@@ -198,7 +198,8 @@ export async function formatPricesForProduct(
         noContextOptions,
       )
 
-    const {id, ...merchantCouponWithoutIdentifier} = appliedMerchantCoupon
+    const {identifier, merchantAccountId, ...merchantCouponWithoutIdentifier} =
```

Comment:
this is what was causing the issue. in our app `identifier` is the Stripe coupon code ID and we are sanitizing it before sending it to the client, but I changed this to `id` at some point and that broke the coupon getting sent to stripe properly for PPP

added tests above